recipes/ffmpeg/all/conanfile.py

```diff
@@ -198,12 +198,15 @@ def _target_arch(self):
 
     @property
     def _target_os(self):
-        _, _, target_os = tools.get_gnu_triplet(
-            "Macos" if tools.is_apple_os(self.settings.os) else str(self.settings.os),
-            str(self.settings.arch),
-            str(self.settings.compiler) if self.settings.os == "Windows" else None,
-        ).split("-")
-        return target_os
+        if self.settings.os == "Windows":
+            return "win32"
+        else:
+            _, _, target_os = tools.get_gnu_triplet(
+                "Macos" if tools.is_apple_os(self.settings.os) else str(self.settings.os),
+                str(self.settings.arch),
+                str(self.settings.compiler) if self.settings.os == "Windows" else None,
+            ).split("-")
+            return target_os
```

Comment:
An manual mapping is probably required: https://github.com/FFmpeg/FFmpeg/blob/9dd104f6e263dce770541074d99b5286524f2a8b/configure#L5382-L5640

Reply:
MinGW support can be added later, on demand